tests/integration_tests/allocation_entries/test_get.py

```
@@ -0,0 +1,54 @@
+
+        self.assert_get_all_response(rest_response, soap_response)
+
+    def assert_get_all_response(self, rest_response: list, soap_response: list) -> None:
+        """
+        Assert the get all response.
+        :param rest_response: REST response
+        :param soap_response: SOAP response
+        :return: None
+        """
+        soap_response = list(soap_response)
+        assert len(rest_response['lines']) == len(soap_response[0]), f"Length of REST and SOAP lines do not match: {len(rest_response['lines'])} != {len(list(soap_response)[0])}"
```

Ruff (0.14.3)

Prefer `next(iter(soap_response))` over single element slice

Replace with `next(iter(soap_response))`

(RUF015)